apps/src/templates/lessonOverview/LessonOverview.jsx

```diff
@@ -345,6 +364,9 @@ class LessonOverview extends Component {
   }
 }
 
+// Get the current locale.
```

Comment:
I'm curious if this component could work with the localesRedux? would be nice to standardize the provider of the locale code.

Reply:
Agree on this. Do you have an example of the `localesRedux` being used?

Reply:
currentLocale function also works. I'm not sure the locale redux is initialized on these pages anyway.